docs-src/docs/openlabel/openlabel-format.md

```diff
@@ -285,6 +285,10 @@ The value `MODE_POLY2D_ABSOLUTE` is the only supported value for `mode`. Absolut
 
 
 ## Representing curves
+:::caution
+The name of the interpolation method has changed from `interpolation-method` to `interpolation_method`. However, old annotations
+might still contain the old name
```

Comment:
Might be good to add the version where it changed here.

Reply:
It didn't actually change yet so I don't have a date yet. Will add a date when Alex has changed it though